Source/Core/Core/IOS/ES/ES.cpp

```diff
@@ -173,14 +173,39 @@ static bool UpdateUIDAndGID(const IOS::ES::TMDReader& tmd)
   return true;
 }
 
+static ReturnCode CheckIsAllowedToSetUID(const u32 caller_uid)
```

Comment:
I think this one is more complex on hardware (AHBPROT etc?) but it should be good enough for now I guess...